projects/1.20-fabric/assets/create-food/createfood/lang/zh_cn.json

```
@@ -0,0 +1,1729 @@
+  "block.createfood.raw_berry_pie": "甜浆果派生坯",
+  "block.createfood.raw_cheese_pizza": "奶酪披萨生坯",
+  "block.createfood.raw_cheesecake": "奶酪蛋糕生坯",
+  "block.createfood.raw_chocolate_graham_cracker_pie_crust": "巧克力全麦粉饼干馅饼酥皮生坯",
+  "block.createfood.raw_chocolate_pie": "巧克力派生坯",
+  "block.createfood.raw_chocolate_pie_graham_cracker": "巧克力派生坯（全麦粉饼干）",
+  "block.createfood.raw_chorus_fruit_cheesecake": "紫颂果奶酪蛋糕生坯",
+  "block.createfood.raw_chorus_fruit_pie": "紫颂果派生坯",
+  "block.createfood.raw_cream_pie_chocolate_graham_cracker": "奶油派生坯（巧克力全麦粉饼干）",
+  "block.createfood.raw_cream_pie_graham_cracker": "奶油派生坯（全麦粉饼干）",
+  "block.createfood.raw_fish_bacon_pizza": "鱼肉&培根披萨生坯",
+  "block.createfood.raw_fish_onion_pizza": "鱼肉&洋葱披萨生坯",
+  "block.createfood.raw_fish_pizza": "鱼肉披萨生坯",
+  "block.createfood.raw_glow_berry_cheesecake": "发光浆果奶酪蛋糕生坯",
+  "block.createfood.raw_glow_berry_pie": "发光浆果派生坯",
+  "block.createfood.raw_graham_cracker_pie_crust": "全麦粉饼干馅饼酥皮生坯",
+  "block.createfood.raw_mushroom_bacon_pizza": "蘑菇&培根披萨生坯",
+  "block.createfood.raw_mushroom_fish_pizza": "蘑菇&鱼肉披萨生坯",
+  "block.createfood.raw_mushroom_onion_pizza": "蘑菇&洋葱披萨生坯",
+  "block.createfood.raw_mushroom_pizza": "蘑菇披萨生坯",
+  "block.createfood.raw_onion_bacon_pizza": "洋葱&培根披萨生坯",
+  "block.createfood.raw_onion_pizza": "洋葱披萨生坯",
+  "block.createfood.raw_pie_crust": "馅饼酥皮生胚",
+  "block.createfood.raw_sausage_bacon_pizza": "香肠&培根披萨生坯",
+  "block.createfood.raw_sausage_fish_pizza": "香肠&鱼肉披萨生坯",
+  "block.createfood.raw_sausage_mushroom_pizza": "香肠&蘑菇披萨生坯",
+  "block.createfood.raw_sausage_onion_pizza": "香肠&洋葱披萨生坯",
+  "block.createfood.raw_sausage_pizza": "香肠披萨生坯",
+  "block.createfood.red_gelatin_dessert_block": "红色明胶甜品块",
+  "block.createfood.red_gelatin_mix_block": "红色明胶混合流体",
+  "block.createfood.sausage_bacon_pizza": "香肠&培根披萨",
+  "block.createfood.sausage_fish_pizza": "香肠&鱼肉披萨",
+  "block.createfood.sausage_mushroom_pizza": "香肠&蘑菇披萨",
+  "block.createfood.sausage_onion_pizza": "香肠&洋葱披萨",
+  "block.createfood.sausage_pizza": "香肠披萨",
+  "block.createfood.slime_block": "黏液流体",
+  "block.createfood.smores_pie": "棉花糖夹心饼干派",
+  "block.createfood.sour_cream_block": "酸奶油流体",
+  "block.createfood.squid_ink_block": "鱿鱼墨汁",
+  "block.createfood.sugar_cane_juice_block": "甘蔗汁流体",
+  "block.createfood.toffee_block": "太妃糖流体",
+  "block.createfood.toffee_fudge_block": "太妃软糖流体",
+  "block.createfood.ube_cake_base": "香芋蛋糕坯",
+  "block.createfood.ube_cream_frosted_ube_cake": "香芋奶油糖霜香芋蛋糕",
+  "block.createfood.ube_cream_frosting_block": "香芋奶油糖霜流体",
+  "block.createfood.vegetable_oil_block": "植物油",
+  "block.createfood.vinegar_block": "醋流体",
+  "block.createfood.white_chocolate_block": "白巧克力流体",
+  "block.createfood.white_chocolate_fudge_block": "白巧克力软糖流体",
+  "block.createfood.yellow_gelatin_dessert_block": "黄色明胶甜品块",
+  "block.createfood.yellow_gelatin_mix_block": "黄色明胶混合流体",
+  "block.createfood.yogurt_block": "酸奶",
+  "block.minecraft.cake": "蛋糕（甜浆果）",
+  "item.createfood.apple_cheesecake_slice": "苹果奶酪蛋糕片",
```

```suggestion
  "item.createfood.apple_cheesecake_slice": "苹果奶酪蛋糕切片",
```
制作方法如果和农夫乐事一致，建议按照农夫乐事的“切片”，如改通查